Content.Server/GameObjects/Components/Nutrition/DrinkComponent.cs

```diff
@@ -84,14 +85,24 @@ public override void Initialize()
                 }
             }
 
-            _contents.MaxVolume = _initialContents.TotalVolume;
+            if (Owner.TryGetComponent(out CanSpillComponent canSpillComponent))
```

Comment:
I feel like this would be better served with an event on the solution itself rather than specifically coding for the spill component.